scripts/Deepcell_Postprocessing.ipynb

```diff
@@ -165,9 +172,10 @@
    "outputs": [],
    "source": [
     "# load TIFFs of segmentation labels (from pixel clustering, deepcell, etc.)\n",
-    "segmentation_labels_preload = data_utils.tiffs_to_xr_labels(tiff_dir=label_dir, output_dir=None, delimiter='_')\n",
-    "plt.figure(figsize = (13, 13))\n",
-    "plt.imshow(segmentation_labels_preload.loc[\"Point8\", :, :, \"segmentation_label\"])"
+    "if not legacy_deepcell:\n",
+    "    segmentation_labels = data_utils.tiffs_to_xr_labels(tiff_dir=deepcell_dir, output_dir=None, delimiter='_')\n",
```

Comment:
I think we should change the delimiter to be` _feature_0.tif`. This will be robust to people having underscores in their FOV names